homework5/tests/test_courses.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from homework5.task1.courses import Homework, Student, Teacher
+
+
+@pytest.mark.parametrize(
+    ["data", "expected_result"], [(Student("Roman", "Petrov"), True)]
+)
+def test_student_init(data: Student, expected_result: bool):
```

Comment:
what's the purpose of "expected_result" parameter? Also, what's the purpose of using parametrize fixture if there are only 1 group of values?